admin/src/App.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div id="app"></div>
+</template>
+
+<script>
+export default {
+  name: 'App',
+  methods: {
+    async readToken() {
+      if (window.localStorage && window.localStorage.getItem('vuex')) {
+        this.$store.commit('token', JSON.parse(window.localStorage.vuex).token)
+      }
+    },
+  },
+  created() {
+    this.readToken()
+  },
```

Comment:
see `router/guards.js` for a suggestion. We also must assume we cannot access the coresponding localStorage entry (see other comment of mine) and we actually should ensure nobody but our domain can.

```suggestion
```